drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -849,7 +849,18 @@ void RasterizerCanvasGLES3::_record_item_commands(const Item *p_item, RID p_rend
 
 		if (c->type != Item::Command::TYPE_MESH) {
 			// For Meshes, this gets updated below.
-			_update_transform_2d_to_mat2x3(base_transform * draw_transform, state.instance_data_array[r_index].world);
+			Transform2D final_transform = base_transform * draw_transform;
+			switch (p_item->texture_filter) {
```

Comment:
Given Calinou's question I'd say a comment here explaining this would be helpful 🙂